packages/common/src/node-status-interface.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * A JSON object containing various diagnostic and introspection information from a running
+ * Ceramic node.
+ */
+export interface NodeStatusResponse {
+  /**
+   * A random UUID that is generated each time a node starts up.  Can be used to detect when
+   * a node restarts.
+   */
+  runId: string
+
+  /**
+   * The Ceramic network the node is connected to.
+   */
+  network: string
+
+  /**
+   * Anchoring-related information.
+   */
+  anchor: AnchorNodeStatus
+
+  /**
+   * Information about the connected IPFS node.
+   */
+  ipfs: IpfsNodeStatus
+}
+
+/**
+ * IPFS sub-object within NodeStatusResponse
+ */
+export interface IpfsNodeStatus {
+  /**
+   * PeerId of the connected ipfs node
+   */
+  peerId: string
+
+  /**
+   * IPFS Swarm multiaddrs of the connected ipfs node
+   */
+  addresses: Array<string>
+}
+
+export interface AnchorNodeStatus {
+  /**
+   * The URL of the Ceramic Anchor Service used to request anchors.
+   */
+  anchorServiceUrl: string
+
+  /**
+   * The endpoint used to validate anchor transactions (generally this is an ethereumRpcEndpoint).
+   */
+  anchorValidationEndpoint: string | null
```

Comment:
Can we just call this `ethereumRpcEndpoint` for clarity?

Reply:
Yeah we can.  We have all these interfaces and indirection layers around the anchor service and anchor validator that are designed to make it blockchain agnostic and able to support other chains in the future, so exposing the ethereumRpcEndpoint in the generic AnchorValidator class felt like a layer violation.  But if realistically we're never going to support another chain I can go ahead and do that.

Reply:
Having a Ceramic node validate anchors on multiple chains seems problematic since this would require a node operator to run a lot of different nodes. I think for simplicity and decentralization we should stick to using one chain per ceramic network.

By decentralization here I mean that if you need to run chain1-node, chain2-node, and ceramic-node, there is less people that can feasibly do this which reduces the decentralization of the network.

Reply:
okay will change